var/ramble/repos/builtin/applications/wrfv4/application.py

```diff
@@ -23,10 +23,10 @@ class Wrfv4(SpackApplication):
                   variants='build_type=dm+sm compile_type=em_real nesting=basic ~chem ~pnetcdf',
                   compiler='gcc9', mpi='impi2018', required=True)
 
-    input_file('CONUS_2p5km', url='https://www2.mmm.ucar.edu/wrf/users/benchmark/v42/v42_bench_conus2.5km.tar.gz',
+    input_file('CONUS_2p5km', url='https://www2.mmm.ucar.edu/wrf/users/benchmark/v422/v42_bench_conus2.5km.tar.gz',
```

Comment:
Could you also please update the links in the tests (test_wrfv4_dry_run) so they match? (See the CI output for details)